packages/jest-dev-server/package.json

```diff
@@ -12,18 +12,18 @@
     "server"
   ],
   "scripts": {
-    "prebuild": "rm -rf lib/",
+    "prebuild": "rimraf lib/",
     "build": "babel --config-file ../../babel.config.js -d lib --ignore \"**/*.test.js\" src",
     "dev": "yarn build --watch",
     "prepublishOnly": "yarn build"
   },
   "dependencies": {
-    "chalk": "^3.0.0",
+    "chalk": "^4.1.0",
     "cwd": "^0.10.0",
-    "find-process": "^1.4.3",
-    "prompts": "^2.3.0",
+    "find-process": "^1.4.4",
+    "prompts": "^2.4.1",
     "spawnd": "^4.4.0",
     "tree-kill": "^1.2.2",
-    "wait-on": "^4.0.0"
+    "wait-on": "^5.3.0"
```

Comment:
This is most likely a breaking change since `wait-on` uses `axios` instead of `request` in v5